plugins/kotlin/jvm-debugger/test/test/org/jetbrains/kotlin/idea/debugger/test/KotlinInlineCallStackTest.kt

```diff
@@ -0,0 +1,2537 @@
+// Copyright 2000-2022 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.kotlin.idea.debugger.test
+
+import org.jetbrains.kotlin.idea.debugger.test.mock.MockMethodInfo
+
+@Suppress("SpellCheckingInspection")
+class KotlinInlineCallStackTest : AbstractLocalVariableTableTest() {
```

Comment:
Today we've discussed the issue. We have come to an agreement that it will be nice to run the debugger tests on an android emulator. And it looks like our QA team is also working on it, so we are waiting for their status updates. The only drawback here is that it will be hard to fix these tests once they fail because of the emulator, but we'll see how it goes.

Reply:
I've uploaded the generator under [sfs/kotlin-debug-info-extractor](https://github.com/sfs/kotlin-debug-info-extractor)